src/ast/mod.rs

```diff
@@ -4054,6 +4054,12 @@ pub enum Statement {
         arguments: Vec<Expr>,
         options: Vec<RaisErrorOption>,
     },
+    /// Go (MSSQL)
+    ///
+    /// GO is not a Transact-SQL statement; it is a command recognized by various tools as a batch delimiter
+    ///
+    /// See <https://learn.microsoft.com/en-us/sql/t-sql/language-elements/sql-server-utilities-statements-go>
```

Comment:
For visibility we can move the comment to the struct definition similar to the print statement?

Reply:
Ah indeed, ideally both would have the doc comments on their respective structs, I think I meant to reference return  statement in the other PR as the example